src/renderer/containers/map/infoBox/index.jsx

```diff
@@ -32,13 +32,20 @@ export default class InfoBox extends React.Component {
     const address = `https://www.google.com/maps/place/${conversion(
       details.address1
     )}+${details.zipCode}`;
+    const email = {
```

Comment:
Thought about this, but it might be interesting we created an email account for the application so we could be CC'd on the emails being reported by the users. Did not add that here because I wanted thoughts from the group on that. Please respond to this comment or bring it up at the meetup in a couple weeks.

Reply:
I think the answer to this might be tied to who would be responsible for monitoring the incoming emails and what would be their responsibility (even if it's just monitoring them)? Are there any downsides to adding an email account? I can't think of any offhand, but we should consider it.

Reply:
I vote to add an email for our own stats and record keeping as app provider.

Reply:
added a cc email.